content/guides/delivering-deployments.md

```diff
@@ -162,7 +162,7 @@ to simulate work that's going on. During that processing, we also make a call to
 switch the status to `pending`.
 
 After the deployment is finished, we set the status to `success`. You'll notice
-that this pattern is the exact same as when we you your CI statuses.
+that this pattern is the exact same as those which we display in your CI statuses.
```

Comment:
:wave: Hi @vmaudgalya, thank you for the pull request. 

I conferred with the original author of this guide and we believe that the second sentence in this paragraph referred to a previous iteration of the guide which included CI as a topic. Since then, we've moved over to a separate guide for [Building a CI Server](https://developer.github.com/guides/building-a-ci-server/).

That being said, I think we can completely remove this second sentence which will get rid of the typo:

> After the deployment is finished, we set the status to `success`. ~~You'll notice that this pattern is the exact same as those which we display in your CI statuses.~~

If you'd like to update this pull request to include that change, I'd be happy to merge it.